### ISSUE TITLE
PHP 8.2 Support

### ISSUE THREAD

**Issue by TheRealKingS** [Edugate/Jagger#314]
As far as i can tell, the code works with 8.1 - will 8.2 be official supported?

**Comment by servicesarchitecturemanager** [comment]
no, Jagger will be replaced with an alternative based on Go in 2025

**Comment by malavolti** [comment]
Hi @servicesarchitecturemanager ,

Do you have news regarding the Go version of Jagger?

Thank you.

**Comment by filipesantana18** [comment]
Hello,
Is there any prediction of the new Jagger in Go?

**Comment by janul** [comment]
we're almost there :)
